scriptmodules/supplementary/bluetooth.sh

```diff
@@ -46,33 +46,53 @@ function get_script_bluetooth() {
     echo "$name"
 }
 
+function bluez_cmd() {
```

Comment:
function naming is wrong - needs to include _bluetooth so it's part of the module.